openldap/server.sls

```diff
@@ -7,8 +7,8 @@
   file.managed:
     - source: salt://openldap/files/slapd.conf
     - template: jinja
-    - user: root
-    - group: {{ openldap.su_group }}
+    - user: {{ openldap.slapd_user }}
+    - group: {{ openldap.slapd_group }}
```

Comment:
Just confirming, is this change only needed here?  The other places where this occurs:

https://github.com/saltstack-formulas/openldap-formula/blob/9f758a4a599bb521b2b40cc9b4eab249c53bb4ec/openldap/server.sls#L19-L24

https://github.com/saltstack-formulas/openldap-formula/blob/9f758a4a599bb521b2b40cc9b4eab249c53bb4ec/openldap/server.sls#L35-L38

https://github.com/saltstack-formulas/openldap-formula/blob/ae96c9f1e301e3bd7d10fb7538c9a18113ab28a7/openldap/client.sls#L2-L10

Reply:
`default` files are generally owned by `root`.
On the `client` conf . it is not necessary neither